pkg/core/helper_test.go

```diff
@@ -210,7 +210,7 @@ func TestCreateBasicChain(t *testing.T) {
 	t.Logf("txMoveNeo: %s", txMoveNeo.Hash().StringLE())
 	t.Logf("txMoveGas: %s", txMoveGas.Hash().StringLE())
 
-	require.True(t, util.Fixed8FromInt64(1000).CompareTo(bc.GetUtilityTokenBalance(priv0ScriptHash)) <= 0)
+	require.True(t, bc.GetUtilityTokenBalance(priv0ScriptHash) >= 100000000000)
```

Comment:
It's hard to count the number of zeroes here.

Reply:
Fixed by using `native.GASFactor`.